trinity/protocol/eth/validators.py

```diff
@@ -0,0 +1,107 @@
+from typing import (
+    Dict,
+    Tuple,
+)
+
+from eth.rlp.headers import BlockHeader
+from eth.rlp.receipts import Receipt
+from eth_typing import (
+    Hash32,
+)
+from p2p.exceptions import (
+    ValidationError,
+)
+
+from trinity.rlp.block_body import BlockBody
+from trinity.protocol.common.validators import (
+    BaseValidator,
+    BaseBlockHeadersValidator,
+)
+
+from . import constants
+
+
+class GetBlockHeadersValidator(BaseBlockHeadersValidator):
+    protocol_max_request_size = constants.MAX_HEADERS_FETCH
+
+
+NodeDataBundles = Tuple[Tuple[Hash32, bytes], ...]
```

Comment:
These should probably be part of the shared ones from `trinity.protocol.common.types` since the `les` protocol has matching commands.